CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Change log
+
+## Release 2.3.0
+
+Changes:
+
+1. New vaccines:
+    1. Abdala, used in Vietnam
+    2. WIBP-CorV, used in Vietnam
+2. New manufacturers:
+    1. Center for Genetic Engineering and Biotechnology (CIGB) for Abdala
+    2. Sinopharm - Wuhan Institute of Biological Products for WIBP-CorV
+
+The [guidelines document v1.5 has been adopted and published](https://ec.europa.eu/health/sites/default/files/ehealth/docs/digital-green-value-sets_en.pdf) through the [eHealth network page on the EU DCC](https://ec.europa.eu/health/ehealth/covid-19_en).
+
+
+## Prior releases
+
+No change log kept for releases before 2.3.0.
```

Comment:
@dslmeinte There is a changelog captured in the `releases`

- https://github.com/ehn-dcc-development/ehn-dcc-valuesets/releases/tag/2.2.0
- https://github.com/ehn-dcc-development/ehn-dcc-valuesets/releases/tag/2.1.0 
- https://github.com/ehn-dcc-development/ehn-dcc-valuesets/releases/tag/2.0.0

I'm not suggesting to put transfer it into the `CHANGELO.md`. 

We may just refer to them or add the links.

Reply:
I'm very much in favour of making the changelog part of the commits, since that info on the releases page can be edited outside of versioning, and is only available there. It's OK to _also_ put it there, but the change to `CHANGELOG.md` should then be authoritative.

Agree?